base/errorshow.jl

```diff
@@ -417,12 +417,8 @@ function show_method_candidates(io::IO, ex::MethodError, @nospecialize kwargs=()
                         end
                     end
                 end
-                kwords = Symbol[]
-                if isdefined(ft.name.mt, :kwsorter)
-                    kwsorter_t = typeof(ft.name.mt.kwsorter)
-                    kwords = kwarg_decl(method, kwsorter_t)
-                    length(kwords) > 0 && print(iob, "; ", join(kwords, ", "))
-                end
+                kwords = kwarg_decl(method)
+                length(kwords) > 0 && print(iob, "; ", join(kwords, ", "))
```

Comment:
```suggestion
                if length(kwords) > 0
                    print(iob, "; ")
                    join(iob, kwords, ", ")
                end
```

(for better performance and sometimes correctness)